specification/maps/data-plane/Weather/preview/1.0/weather.json

```diff
@@ -834,6 +839,7 @@
           {
             "name": "windowGeometry",
             "description": "When true, window geometry data (geoJSON) is included in the response",
+            "x-ms-client-name": "include_window_geometry",
```

Comment:
Camel casing? `includeWindowGeometry`

Reply:
Other `x-ms-client-name` should also be camelCase.

Reply:
Updated